src/playbacks/hls/hls.js

```diff
@@ -119,6 +133,22 @@ export default class HLS extends HTML5VideoPlayback {
     this.trigger(Events.PLAYBACK_SETTINGSUPDATE)
   }
 
+  onError(evt, data) {
+    if (data && data.fatal) {
+      switch (data.type) {
+        case Hls.ErrorTypes.NETWORK_ERROR:
+          hls.startLoad()
```

Comment:
@tjenkinson: Isn't there a timeout on the request itself already? Yes, a retry limit might be a good idea.